### PR TITLE
update: translate seata-raft-detailed-explanation.md to english 

### LINKED ISSUE #808: task: translate the blog “seata-raft-detailed-explanation.md” to English

### LINKED ISSUE #810: task: translate the blog “seata-1.5.2.md” to English

### PR DESCRIPTION
**Pull request by lightClouds917**
## What is the purpose of the change

update: translate seata-raft-detailed-explanation.md to english 
update: translate the blog “seata-1.5.2.md” to English

fixes #808
fixes #810 

## Brief changelog

XXXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [GITHUB_issue](https://github.com/apache/incubator-seata-website/issues) filed for the change(usually before you start working on it).  Trivial changes such as typos do not necessitate a GITHUB issue. Your pull request should solely focus on addressing this specific issue without incorporating any other modifications. Each PR should resolve a single issue.
- [x] Format the pull request title like `doc: add an operations guide document`. Each commit within the pull request should contain a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Test your code locally by running `npm run build`, and make sure it works as expected.
- [x] Ensure that no files under the `build` folder are included in the commit, as well as the `package-lock.json` and `yarn.lock` files.

### CODE REVIEW

File: i18n/en/docusaurus-plugin-content-blog/seata-raft-detailed-explanation.md [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+
+However, the prerequisite is that computation and storage must be separated. Why is the integration of computation and storage not feasible? This brings us to the implementation of the File mode. As described earlier, the File mode stores data on local disks and node memory, with no synchronization in data writing operations. This means that the current File mode cannot achieve high availability and only supports single-machine deployment. For basic quick start and simple use, the File mode has lower applicability, and the high-performance, memory-based File mode is practically no longer used in production environments.
+
+## 2.3 How is Seata-Raft Designed?
+
+### 2.3.1 Design Principles
+
+The design philosophy of Seata-Raft mode is to encapsulate the File mode, which is unable to achieve high availability, and use the Raft algorithm to synchronize data between multiple TCs. This mode ensures data consistency among multiple TCs when using the File mode and replaces asynchronous flushing operations with Raft logs and snapshots for data recovery.
+
+![flow](https://blog.funkye.icu/img/blog/Dingtalk_20230105203431.jpg)
```
Comment (slievrly) [inline i18n/en/docusaurus-plugin-content-blog/seata-raft-detailed-explanation.md:69]: All image links need to be replaced with internal resource addresses.

File: i18n/en/docusaurus-plugin-content-blog/seata-raft-detailed-explanation.md [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+- [4. Benchmark Comparison](#)
+- [5. Conclusion](#)
+
+# 1. Overview
+
+Seata is an open-source distributed transaction solution with over 24000 stars and a highly active community. It is dedicated to providing high-performance and user-friendly distributed transaction services in microservices architecture.
+
+Currently, Seata's distributed transaction data storage modes include file, db, and redis. This article focuses on the architecture, deployment and usage, benchmark comparison of Seata-Server Raft mode. It explores why Seata needs Raft and provides insights into the process from research and comparison to design, implementation, and knowledge accumulation.
+
+Presenter: Chen Jianbin (funkye) github id: [funky-eyes](https://github.com/funky-eyes)
```
Comment (funky-eyes) [inline i18n/en/docusaurus-plugin-content-blog/seata-raft-detailed-explanation.md]:
```suggestion
Presenter: Jianbin  Chen(funkye) github id: [funky-eyes](https://github.com/funky-eyes)
```

File: i18n/en/docusaurus-plugin-content-blog/seata-raft-detailed-explanation.md (hunk 1 above)
Comment (lightClouds917) [reply i18n/en/docusaurus-plugin-content-blog/seata-raft-detailed-explanation.md:69]: ok，i will replace

**Review by funky-eyes — APPROVED** [review]
LGTM

**Pull request merged**